libs/langchain_v1/langchain/agents/middleware/summarization.py

```diff
@@ -119,6 +120,12 @@
 """
 
 
+def _get_approximate_token_counter(model: BaseChatModel) -> TokenCounter:
```

Comment:
Could you add a comment explaining what's going on here? This code isn't self-explanatory. Where is 3.3 from?